plugins/kubernetes-backend/src/routes/resourceRoutes.test.ts

```diff
@@ -228,23 +222,27 @@ describe('resourcesRoutes', () => {
         });
     });
     // eslint-disable-next-line jest/expect-expect
-    it('401 when no Auth header', async () => {
+    it('403 when no Auth header', async () => {
       await request(app)
         .post('/api/kubernetes/resources/workloads/query')
+        .set('authorization', mockCredentials.none.header())
```

Comment:
slightly icky :) could have initialized with a default credentials of none on the httpAuth instead right